app/views/searches/_form.html.erb

```diff
@@ -3,10 +3,11 @@
     controllerName: Settings.filters_beta.present? ? controller_name.to_json : nil,
     advancedSearchText: (Settings.filters_beta.present? ? @advanced_text : params[:search]) || "",
     allForms: @all_forms || [],
-    selectedFormIds: @form_ids || [],
-    isReviewed: nil,
-    selectedUsers: Settings.filters_beta.present? ? [] : [],
-    selectedGroups: Settings.filters_beta.present? ? [] : [],
+    selectedFormIds: @selected_form_ids || [],
+    selectedQings: @selected_qings || [],
+    isReviewed: @is_reviewed,
+    selectedUsers: @selected_users || [],
+    selectedGroups: @selected_groups || [],
```

Comment:
I wonder if a cleaner way to do this would be to have a serializer for the ResponsesSearcher class.  ResponsesSearcher would then be responsible for all of this. allForms could be added, advancedSearch will be there eventually anyway, controllerName could just be hardcoded to responses. Seems to make sense, no?